Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -369,7 +369,6 @@ public FilteredIdentity FilteredIdentity(Symbol symbol, TimeSpan resolution, Fun
         /// <param name="period">The period of the FRAMA</param>
         /// <param name="longPeriod">The long period of the FRAMA</param>
         /// <param name="resolution">The resolution</param>
-        /// <param name="selector">Selects a value from the BaseData to send into the indicator, if null defaults to the Value property of BaseData (x => x.Value)</param>
         /// <returns>The FRAMA for the given parameters</returns>
         public FractalAdaptiveMovingAverage FRAMA(Symbol symbol, int period, int longPeriod = 198, Resolution? resolution = null)
```

Comment:
Actually this warning is pointing to a bug here :)
All indicator helper methods have an optional selector parameter, we should add the missing parameter and leave the XML comment.